nemo/collections/tts/data/dataset.py

```diff
@@ -483,6 +490,15 @@ def add_energy(self, **kwargs):
     def add_speaker_id(self, **kwargs):
         pass
 
+    def add_gst_ref_audio(self, **kwargs):
```

Comment:
add_gst_ref_audio -> add_reference_audio

Reply:
Add comment/description of what this method is doing, what does self.speaker_to_index represent.